e2e/components/menu/menu.e2e.ts

```diff
@@ -12,7 +12,7 @@ describe('menu', () => {
     page.trigger().click();
```

Comment:
Have you found it generally more difficult to add unit tests for the menu than e2e tests? The e2e tests are good at capturing the "real" behavior, but they take much longer to run (and are flakier), so covering as much of the behavior as possible in unit tests will generally make maintenance much easier later.